server/src/main/java/io/crate/execution/engine/aggregation/impl/MaximumAggregation.java

```diff
@@ -87,7 +87,7 @@ public LongMax(String columnName, DataType<?> partialType) {
 
         @Override
         public MutableLong initialState() {
-            return new MutableLong();
+            return new MutableLong(Long.MIN_VALUE);
```

Comment:
The value defaulted to 0, so if the table only contained negative values the result would have been `null`, because each value would have been smaller than 0 and it would have assumed that there was no value at all.

No change entry or so because the max aggregate change wasn't released yet.